netctl/config.go

```diff
@@ -50,8 +50,19 @@ func configExists(ctx *cli.Context) bool {
 
 // configPath returns the full path to the user's netctl config file
 func configPath() string {
+	var homeDir string
+	// this fails for static binaries
 	usr, err := user.Current()
+	if err == nil {
+		homeDir = usr.HomeDir
+	}
+	// this should work where we don't have static binaries
 	if err != nil {
+		homeDir = os.Getenv("HOME")
+	}
+	// panic if we've failed to retrieve the home directory
+	if homeDir == "" {
+		err = fmt.Errorf("user.Current is unavailable & HOME isn't set")
```

Comment:
This error msg doesn't help end user understand whats the problem and how to resolve it, maybe just saying `Failed to determine home directory of current user, please set environment variable HOME`